core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -133,19 +138,30 @@ export class Question {
     return null;
   }
 
-  getUnaddressedMisconceptionNames(misconceptionsBySkill: {}): string[] {
+  getUnaddressedMisconceptionNames(
+      misconceptionsBySkill: Record<string, Misconception[]>
+  ): string[] {
```

Comment:
```suggestion
  getUnaddressedMisconceptionNames(
    misconceptionsBySkill: Record<string, Misconception[]>
  ): string[] {
```

Reply:
show me `Expected indentation of 6 spaces but found 4.`